blockonomics-woocommerce.php

```diff
@@ -501,7 +501,6 @@ function blockonomics_create_table() {
         crypto varchar(3) NOT NULL,
         address varchar(191) NOT NULL,
         timestamp int,
```

Comment:
What is the use of `timestamp` column now?  If its not being used anymore better to remove this also

Reply:
true, now the `timestamp` has been dropped